apps/wing/src/cli.ts

```diff
@@ -165,6 +165,10 @@ async function main() {
     .option("-p, --plugins [plugin...]", "Compiler plugins")
     .option("-r, --rootId <rootId>", "App root id")
     .option("--no-clean", "Keep build output")
+    .option(
+      "-o, --output-file <outputFile>",
+      "File name to write test results to (no file extension in needed)"
+    )
```

Comment:
@tsuf239 WDYT about requiring the extension instead?
I think it would avoid confusion about the output and open the option to use other extensions to designate the output type.

Reply:
ok :)